deposit/resource.ts

```diff
@@ -7,16 +9,19 @@ export type ResourceLicense = {
 	spdxIdentifier: string;
 };
 
-export type ResourceKind =
-	| 'Book'
-	| 'BookChapter'
-	| 'Journal'
-	| 'JournalIssue'
-	| 'JournalArticle'
-	| 'Conference'
-	| 'ConferenceProceeding'
-	| 'ConferencePaper'
-	| 'Other';
+export const resourceKinds = [
+	'Book',
+	'BookChapter',
+	'Journal',
+	'JournalIssue',
+	'JournalArticle',
+	'Conference',
+	'ConferenceProceeding',
+	'ConferencePaper',
+	'Other',
+] as const;
+
+export type ResourceKind = (typeof resourceKinds)[number];
```

Comment:
ive done this a number of times in order to reuse this enum in a schema somewhere